bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/ExitInfoPluginConfiguration.kt

```diff
@@ -11,17 +11,26 @@ class ExitInfoPluginConfiguration(
      */
     var includeLogcat: Boolean = false,
 
+    var reportUnmatchedAnrs: Boolean = true,
+    var reportUnmatchedNativeCrashes: Boolean = true,
```

Comment:
```suggestion
    /**
     * Report [ApplicationExitInfo] ANRs that do not appear to correspond with BugSnag [Event]s
     * as synthesized errors. These will appear on your dashboard without BugSnag data such as
     * breadcrumbs and metadata, but will report crashes that BugSnag is otherwise unable to catch
     * such as background ANRs.
     */
    var reportUnmatchedAnrs: Boolean = true,
    
    /**
     * Report [ApplicationExitInfo] native crashes that do not appear to correspond with BugSnag [Event]s
     * as synthesized errors. These will appear on your dashboard without BugSnag data such as
     * breadcrumbs and metadata, but will report crashes that BugSnag is otherwise unable to catch.
     */
    var reportUnmatchedNativeCrashes: Boolean = true,
```